### PR TITLE
#168781656 view most travelled and all destinations

### PR DESCRIPTION
**Pull request by alainmateso**
#### What does this PR do?
Add top traveled and all destinations on the landing page

#### Description of Task to be completed?
- add top traveled destinations
- add all destinations

#### How should this be manually tested?

- Clone this branch
- run npm i
- run npm run dev
- navigate to localhost:<:port>

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#168781656

#### Screenshots (if appropriate)
Screen Shot 2019-12-26 at 19 26 01
Screen Shot 2019-12-26 at 19 26 59

#### Questions:
N/A

### CODE REVIEW

**Review by Cheza-Dzabala — APPROVED** [review]
Great job bruv

**Review by nignanthomas — CHANGES_REQUESTED** [review]
Great job there @alainmateso 
However, I think those pictures should be links to view all accommodations of a specific location.
Don't you think so?

**Comment by alainmateso** [comment]
> Great job there @alainmateso
> However, I think those pictures should be links to view all accommodations of a specific location.
> Don't you think so?

I believe that could be smoothly implemented after the search accommodation is done. Thus, clicking on a location would search for all accommodations from that location. And there's a feature for that in the pipeline if I'm not mistaken

**Review by nignanthomas — APPROVED** [review]

File: src/components/pages/landingPage/AllDestinations.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      await fetchTopDestinations();
+      await fetchLocations();
+      this.setState({ isLoading: false });
+    }
+
+    render() {
+      const { isLoading } = this.state;
+      const { locations, topDestinations } = this.props;
+      const { data } = locations || {};
+      const { data: topDestinationData } = topDestinations || {};
```
Comment (nakiwuge) [inline src/components/pages/landingPage/AllDestinations.js:27]: Add empty line

**Pull request merged**